acra-censor/acra-censor_implementation.go

```diff
@@ -10,6 +11,9 @@ type AcraCensor struct {
 
 func (acraCensor *AcraCensor) AddHandler(handler QueryHandlerInterface) {
 	acraCensor.handlers = append(acraCensor.handlers, handler)
+	sort.Slice(acraCensor.handlers, func(i, j int) bool {
```

Comment:
as we discussed, priority should be fetched from order in configuration. if rule is on top of list then it has higher priority than next rules. or we should change rule structure and add priority field to provide ability to configure it by user
```
handlers:
  - handler: query_ignore
    priority: 1
    queries:
      - ROLLBACK
      - COMMIT
      - BEGIN
  - handler: query_capture
    priority: 3
    filepath: censor_log
  - handler: blacklist
    priority: 2
    queries:
      - INSERT INTO SalesStaff1 VALUES (1, 'Stephen', 'Jiang');
```
if without priority field, than `query_ignore` than rules should executed in next order `query_ignore` -> `query_capture` -> `blacklist`. If user want to run `blacklist` first then he should place it first in config.